class.jetpack.php

```diff
@@ -1229,7 +1229,7 @@ public static function is_development_version() {
 		 */
 		return (bool) apply_filters(
 			'jetpack_development_version',
-			! preg_match( '/^\d+(\.\d+)+$/', JETPACK__VERSION )
+			! preg_match( '/^\d+(\.\d+)+$/', Jetpack_Constants::get_constant( 'JETPACK__VERSION' ) )
```

Comment:
The value is very much testing. 

One of the things that we need to figure out is better code coverage, which is a pre-requisite for potentially running `master` on Pressable and other hosts. I don't think we have to do it all at the same time. That'd be a nightmare of a PR.

But, as we go through and write tests, we can begin swapping out code like `defined()` with `Jetpack_Constants:is_defined()` and `I_AM_A_COOL_CONSTANT` with `Jetpack_Constants::get_constant( 'I_AM_A_COOL_CONSANT' )`.

Better tests allow us to test our assumptions about the code we write as well as improve confidence in our product for outside users.